tezos-encoding/Cargo.toml

```diff
@@ -1,9 +1,14 @@
 [package]
 name = "tezos_encoding"
```

Comment:
wdyt about renaming this crate to tezos-data-encoding?

Reply:
lets do it!